### PR TITLE
Update security.rst

### PR DESCRIPTION
**Pull request by tabeth**
Someone following the example here will mistakenly think `NODENAME` has been set. From a beginner's perspective I think `_local` is an OK default.

## Overview

From a beginner's perspective having `NODENAME` set will allow them to follow the example more closely. 

## Checklist

- [x] Documentation is written and is accurate;

### CODE REVIEW

**Comment by wohali** [comment]
Hm, ok, except that this means you can't be talking through a load balancer, and you will only ever be able to query the info of the node you're speaking to.

If you want to improve this further, do you want to put in a short description of what `NODENAME` is, where to find it, and that we also support the `_local` alias to mean the specific node you're talking to (pointing out the load balancer problem)?

**Comment by tabeth** [comment]
Good point. I'll incorporate those things as well as link to the relevant items.

**Comment by tabeth** [comment]
Wow, @wohali, apologies. I thought I made these changes months ago! In any case, I've added a link to the Node Management page, as well as a quick explanation to what `_local`  and `NODENAME` do in this context.

File: src/intro/security.rst [hunk 1]
```diff
@@ -99,6 +100,13 @@ As per the :ref:`_config <api/config>` API's behavior, we're getting
 the previous value for the config item we just wrote. Since our admin user
 didn't exist, we get an empty string.
 
+Please note that ``_local`` serves as an  alias for the local node name, so 
+for all configuration URLs, ``NODENAME`` may be set to ``_local``, to interact 
+with the local node’s configuration.
+
+.. seealso::
+    :ref:`Node Management <stable/cluster/node>`
```
Comment (wohali) [inline src/intro/security.rst]: Unfortunately, this isn't a valid reference. Can you please fix this?

**Comment by wohali** [comment]
Hey @tabeth you need to remove the `stable/`, that's just an artefact of how ReadTheDocs publishes our content on docs.couchdb.org for us. (It's a version label - it could be `latest/` or `2.3.1/` or so on.)

Just use `<cluster/nodes>`.

**Comment by tabeth** [comment]
Should be good to go!

**Comment by wohali** [comment]
@tabeth Congratulations on your first contribution to Apache CouchDB! :confetti_ball: :frog: :gem:

**Comment by tabeth** [comment]
@wohali Expect more where that came from!

**Pull request merged**